qemu/tests/usb_common.py

```diff
@@ -112,24 +112,36 @@ def verify_usb_device_in_guest(params, session, devs):
     def _verify_guest_usb():
         output = session.cmd_output(params["chk_usb_info_cmd"],
                                     float(params["cmd_timeout"]))
+        if "Hub" in str(devs) and os_type == "linux":
+            hub_output = session.cmd_output("lsusb -v",
+                                            float(params["cmd_timeout"]))
```

Comment:
Suggest to add a comment for this trick situation, how about ?
```
# For usb-hub, '-v' must be used to get the expected usb info.          
# For non usb-hub, refer to 'chk_usb_info_cmd', two situations:                                               
# '-v' must be used to get expected info                                
# '-v' must not be used to avoid duplicate info in output and affect the device count.
```

Reply:
Updated, thanks!